poi.go

```diff
@@ -466,14 +466,15 @@ func toBranch(branchNames []string) []Branch {
 	results := []Branch{}
 
 	for _, branchName := range branchNames {
-		splitedNames := strings.Split(branchName, ",")
-		head := false
-		if splitedNames[0] == "*" {
-			head = true
-		}
+		splitedNames := strings.Split(branchName, ":")
+
+		headValue := splitedNames[0]
+		refNameValue := splitedNames[1]
+
+		head := headValue == "*"
```

Comment:
Nice that the if statement is gone 🙂
I think this is better because the readability is not much different.

```suggestion
		head := splitedNames[0] == "*"
```